### ISSUE TITLE
Craft 5.5. upgrade fails

### ISSUE THREAD

**Issue by bymayo** [verbb/formie#2139]
### Describe the bug

Trying to update to Craft 5.5. but I'm getting the following error:

```
error: The command "'/usr/bin/php8.2' '/var/www/html/craft' 'migrate/all' '--no-backup' '--no-content'" failed.

Exit Code: 1(General error)

Working directory: /var/www/html

Output:
================


Error Output:
================
PHP Fatal Error 'yii\base\ErrorException' with message 'Class verbb\formie\fields\Address contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (craft\base\PreviewableFieldInterface::previewPlaceholderHtml)'

in /var/www/html/vendor/verbb/formie/src/fields/Address.php:35

Stack trace:
#0 [internal function]: yii\base\ErrorHandler->handleFatalError()
#1 {main}
```

### Steps to reproduce

1. Update to Craft 5.5

### Form settings

N/a

### Craft CMS version

5.4.10.1

### Plugin version

3.0.9

### Multi-site?

No

**Comment by michtio** [comment]
I was just about to post the same issue 😅 

```
> Composer\Config::disableProcessTimeout
> @php craft install/check && php craft clear-caches/all --interactive=0 || exit 0
PHP Fatal Error 'yii\base\ErrorException' with message 'Class verbb\formie\fields\Address contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (craft\base\PreviewableFieldInterface::previewPlaceholderHtml)'
```

**Comment by plcdnl** [comment]
same error here

<img width="1680" alt="image" src="https://github.com/user-attachments/assets/9f6915bf-ccc7-4e36-bf2d-91321dfe02ab">

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.9"`. Will have a release out **very** shortly.

**Comment by engram-design** [comment]
Fixed in [3.0.10](https://github.com/verbb/formie/releases/tag/3.0.10)
